website_sale_qty/static/src/js/website_sale_qty.js

```diff
@@ -0,0 +1,47 @@
+/**
+*    Copyright 2016 LasLabs Inc.
+*    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+**/
+
+odoo.define('website_sale_qty', function(require) {
+    'use strict';
+
```

Comment:
You cannot guarantee the presence of something you did not require. The other modules are wrong if they did not require things before using